sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -91,7 +91,7 @@ class DecisionBoundaryDisplay:
     surface_ : matplotlib `QuadContourSet` or `QuadMesh`
         If `plot_method` is 'contour' or 'contourf', `surface_` is a
         :class:`QuadContourSet <matplotlib.contour.QuadContourSet>`. If
-        `plot_method is `pcolormesh`, `surface_` is a
+        `plot_method is "pcolormesh"`, `surface_` is a
```

Comment:
```suggestion
        `plot_method` is "pcolormesh", `surface_` is a
```
Typo fix.

---

A few lines above 
```
If `plot_method` is 'contour' or 'contourf', `surface_`
```
is used. Should we have consistent use of `'` and `"`?